packages/components/src/QualityBar/QualityBar.component.js

```diff
@@ -1,16 +1,58 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { useTranslation } from 'react-i18next';
-import RatioBar from '../RatioBar';
+import { omit } from 'lodash';
```

Comment:
can you avoid to use omit?

https://levelup.gitconnected.com/omit-is-being-removed-in-lodash-5-c1db1de61eaf